src/Components/Scenes/Chrome/AppBar/AppBar.js

```diff
@@ -91,8 +99,12 @@ const Appbar = props => {
               <KeyboardArrowDownIcon />
             </Box>
           </>
-        ) : (
+        ) : curPath === '/' ? (
```

Comment:
I find nested ternarys difficult to read from another's perspective in general. I prefer to avoid nested ternaries as much as possible but up to you.